src/Util/TestUtil.js

```diff
@@ -168,6 +170,41 @@ export class TestUtil {
 
     return feat;
   })
+
+  /**
+   * Returns a polygon feature with a random position.
+   * @type {Object}
+   */
+  static generatePolygonGeometry = (() => {
+    const polyCoords = [];
+    for (let i =0;i<5;i++){
+      const coords = [
+        Math.floor(Math.random() * 180) - 180,
+        Math.floor(Math.random() * 90) - 90
+      ];
+      polyCoords.push(coords);
+    }
+    const extent= OlExtent.boundingExtent(polyCoords);
+    const geom = new OlGeomPolygon.fromExtent(extent); 
+    return geom;
+  })
+
+  /**
+   * Returns a random extent .
+   * @type {array}
+   */
+  static generateExtent = (() => {
+    const polyCoords = [];
+    for (let i =0;i<5;i++){
```

Comment:
whitespace. we should have a rule enforcing this. can be done in a separate PR

Reply:
New Issue: #522